src/m_config.c

```diff
@@ -688,7 +688,7 @@ void M_LoadDefaults(void)
 
     if ((f = M_fopen(defaultfile, "r")))
     {
-        char s[256];
+        char s[1024];
```

Comment:
I noticed that the strings were getting truncated in woof.cfg so I increased this value. I'm not sure if there are any side effects.